corehq/messaging/scheduling/scheduling_partitioned/tests/test_dbaccessors_non_partitioned.py

```diff
@@ -0,0 +1,245 @@
+import uuid
+from corehq.form_processor.tests.utils import only_run_with_non_partitioned_database
+from corehq.messaging.scheduling.scheduling_partitioned.dbaccessors import (
+    get_alert_schedule_instance,
+    get_timed_schedule_instance,
+    save_alert_schedule_instance,
+    save_timed_schedule_instance,
+    delete_alert_schedule_instance,
+    delete_timed_schedule_instance,
+    get_active_alert_schedule_instance_ids,
+    get_active_timed_schedule_instance_ids,
+    get_alert_schedule_instances_for_schedule,
+    get_timed_schedule_instances_for_schedule,
+)
+from corehq.messaging.scheduling.models import (
+    AlertSchedule,
+    TimedSchedule,
+)
+from corehq.messaging.scheduling.scheduling_partitioned.models import (
+    AlertScheduleInstance,
+    TimedScheduleInstance,
+)
+from corehq.util.exceptions import AccessRestricted
+from datetime import datetime, date
+from django.conf import settings
+from django.test import TestCase
+
+
+@only_run_with_non_partitioned_database
+class BaseSchedulingNontPartitionedDBAccessorsTest(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(BaseSchedulingNontPartitionedDBAccessorsTest, cls).setUpClass()
+        cls.domain = 'scheduling-non-partitioned-test'
+        cls.db = 'default'
+
+    @classmethod
+    def make_alert_schedule_instance(cls, schedule_instance_id=None, schedule_id=None, active=True):
+        return AlertScheduleInstance(
+            schedule_instance_id=schedule_instance_id or uuid.uuid4(),
+            domain=cls.domain,
+            recipient_type='CommCareUser',
+            recipient_id='user-id',
+            current_event_num=0,
+            schedule_iteration_num=1,
+            next_event_due=datetime(2017, 3, 1),
+            active=active,
+            alert_schedule_id=schedule_id or uuid.uuid4(),
+        )
+
+    @classmethod
+    def make_timed_schedule_instance(cls, schedule_instance_id=None, schedule_id=None, active=True):
+        return TimedScheduleInstance(
+            schedule_instance_id=schedule_instance_id or uuid.uuid4(),
+            domain=cls.domain,
+            recipient_type='CommCareUser',
+            recipient_id='user-id',
+            current_event_num=0,
+            schedule_iteration_num=1,
+            next_event_due=datetime(2017, 3, 1),
+            active=active,
+            timed_schedule_id=schedule_id or uuid.uuid4(),
+            start_date=date(2017, 3, 1),
+        )
+
+
+class TestSchedulingNonPartitionedDBAccessorsGetAndSave(BaseSchedulingNontPartitionedDBAccessorsTest):
+
+    def tearDown(self):
+        AlertScheduleInstance.objects.using(self.db).filter(domain=self.domain).delete()
```

Comment:
using is required because it's a subclass of `PartitionedModel` (introduced in this PR) which requires using() to be called on every queryset as a precaution